tests/adf_template_result.json

```diff
@@ -10,7 +10,7 @@
       "content": [
         {
           "type": "text",
-          "text": "Team Stubbe"
+          "text": "Team Stubbe (stubbs)"
```

Comment:
Noen grunn til at stubbe er renavnet til stubbs (gjelder mange steder)

Reply:
For at testene tar høyde for overstyrt teknisk teamnavn.